Sofa/framework/Config/CMakeLists.txt

```
@@ -259,6 +259,12 @@ if (SOFA_TRACY)
     message(STATUS "SOFA is compiled with the Tracy profiler client. Use the Tracy server ${SOFA_TRACY_VERSION}.")
 endif()
 
+
+option(SOFA_SCOPED_ADVANCED_TIMER "Compile SOFA with the scoped advanced timer" ON)
```

Calling "scoped advanced timer" the "default timer" in SOFA looks weird to me, especially for the lambda user.
I would say
```suggestion
option(SOFA_ENABLE_BUILTIN_TIMER "Enable the builtin timers" ON)
```
for example, or something like that.

"Scoped" refer to a C++ thingy and Advanced... I dont know why 😅

---

Yes you are right, I'm not good with names...